src/main/java/Command.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * The Command class makes use of multiple classes to drive the program.
+ */
+public class Command {
+
+    private String command;
+
+    /**
+     * Creates a Command object.
+     *
+     * @param command A string representation of the user's command.
+     */
+    public Command(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Responds accordingly to the user's command.
+     *
+     * @param tasks A TaskList object containing the list of tasks.
+     * @param ui An Ui object that helps to interact with the user.
+     * @param storage A Storage object that contains a file with the tasks in it.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Since the execute method is going to be different for all commands, and there is no Command object that is going to be instantiated (other than its subclasses), its better to make this method (and by extension the class) abstract.